OpenOversight/app/main/views.py

```diff
@@ -801,6 +804,110 @@ def download_dept_csv(department_id):
     return Response(csv, mimetype="text/csv", headers=csv_headers)
 
 
+def check_output(output_str):
+    if output_str == "Not Sure":
+        return ""
+    return output_str
+
+
+@main.route('/download/department/<int:department_id>/officers', methods=['GET'])
+@limiter.limit('5/minute')
```

Comment:
this is an aside, but a good followup here could be to cache the generated CSV

Reply:
yeah, good idea. Do we already have any pages that are cached?

Reply:
I created a separate issue for this: https://github.com/lucyparsons/OpenOversight/issues/742